### PR TITLE
Add Quiz logic

### PR DESCRIPTION
**Pull request by teenoh**

### CODE REVIEW

File: components/challenge/components/Quiz/index.js [hunk 1]
```diff
@@ -42,8 +41,17 @@ const Quiz = ({ challengers, quizItem }) => (
...
+                                    <div>
+                                        <Progress 
+                                            type={circle}
+                                            width={48} 
+                                            // divide by allocated time 
+                                            // for each question (Default: 10s)
+                                            percent={(timeLeft/10) * 100}
+                                            format={() => `${timeLeft}s`} 
+                                        />
+                                    </div>
```
Comment (Way2nnadi) [inline components/challenge/components/Quiz/index.js]: no need for a `div` here

File: pages/admin/challenge/quiz.js [hunk 2]
```diff
@@ -1,33 +1,113 @@
...
+    }
+
+    componentDidMount(){
+        const { quizActive } = this.state;
+        if (!quizActive){
+            this.setState({ questionIndex: 0, quizActive: true  }, () => {
+                this.startTimer();
+            });
+        }
+
```
Comment (Way2nnadi) [inline pages/admin/challenge/quiz.js]: delete unused line

File: pages/admin/challenge/quiz.js [hunk 3]
```diff
@@ -1,33 +1,113 @@
...
+            seconds: 10,
+            
+        };
+
+        this.timer = 0;
+    }
+
+    componentDidMount(){
+        const { quizActive } = this.state;
+        if (!quizActive){
```
Comment (Way2nnadi) [inline pages/admin/challenge/quiz.js:36]: add new line before all `if` statements

**Review by Way2nnadi — APPROVED** [review]
Code looks good to me, only minor syntax issues

**Review by tosinamuda — APPROVED** [review]

**Pull request merged**